_layouts/index.liquid

```diff
@@ -60,9 +60,24 @@ const mailButton = document.getElementById('mail-button');
 const closeButton = document.querySelector('.close-button');
 const modalMessage = document.getElementById('modal-message');
 
-function openModal(message) {
-    modalMessage.innerHTML = message;
-    modal.showModal();
+const userLang = navigator.language;
```

Comment:
I don't love bringing internationalization to JS land when doing static pages. Can we pre-render on the page the possible contents of the modal already translated? We can use [`<template>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template) to have the success and error contents be already on the page but in an "inert state" that avoids loading the images for example.
You can place the templates in under the dialog element,
```html
<dialog>
    <div id="modal-message"></div>
    <template data-msg="success">...</template>
    <template data-msg="error">...</template>
</dialog>
```
Then templates just need to be cloned and added to the dialog, something a bit like this
```js
const message = dialog.querySelector(`[data-msg=${type}]`);
modalMessage.replaceChildren(message.content.cloneNode(true))
```